### DISCUSSION TITLE
подозрение на соежку

### DISCUSSION THREAD

**Discussion by dybtjfy75rg8ig897** [bmax121/APatch#676] [Q&A] [answered]
как определить что к моему телефону подключили программу  или заразили его вирусом который,которая дает возможность доступа к камере микрофону вообщем полному доступу к телефону скрыто

**Comment by Ai686Leo** [comment, marked as answer]
这不属于 APatch 的范围，我建议你到其他地方讨论这个问题
如果你只是想知道应用使用权限的情况，我建议你使用像 App Ops 的第三方权限管理权

Это выходит за рамки APatch, и я предлагаю вам обсудить это в другом месте.
Если вы просто хотите узнать, как ваше приложение использует разрешения, я предлагаю вам использовать сторонний менеджер разрешений, например App Ops.

**Comment by marat2509** [comment]
> как определить что к моему телефону подключили программу или заразили его вирусом который,которая дает возможность доступа к камере микрофону вообщем полному доступу к телефону скрыто

1. Общение только на английском или китайском языках (английский предпочтительнее)
2. Не относиться к APatch
